### PR TITLE
Open source sync 0821

### PR DESCRIPTION
**Pull request by zhangxingmeng**

### CODE REVIEW

File: .antflake8 [hunk 1]
```diff
@@ -0,0 +1,4 @@
+# OPENSOURCE-CLEANUP DELETE_FILE
```
Comment (zhouaihui) [inline .antflake8:1]: Delete this file.

File: requirements.txt [hunk 2]
```diff
@@ -23,12 +23,11 @@ s3fs==2022.1.0
...
 sdc-apis==0.1.0b0; platform_system != "Darwin"  # FEATURE=[lite]
 sdc-sdk==0.1.0b0; platform_system != "Darwin"  # FEATURE=[lite]
-secretflow-rayfed==0.1.1a4 # FEATURE=[lite]
+secretflow-rayfed==0.2.0a7 # FEATURE=[lite]
 # For security reason. This is introduced by TensorFlow.
 setuptools>=65.5.1
 sparse>=0.14.0
---extra-index-url https://test.pypi.org/simple/
-spu==0.4.2b2  # FEATURE=[lite]
+spu 0.5.0.dev20230818  # FEATURE=[lite]
```
Comment (zhouaihui) [inline requirements.txt:30]: `spu 0.5.0.dev20230818`: `==` missed

**Review by zhouaihui — CHANGES_REQUESTED** [review]

**Review by 6fj — APPROVED** [review]
lgtm

**Review by zhouaihui — APPROVED** [review]
LGTM

**Pull request merged**